pkg/webhook/webhook.go

```diff
@@ -53,7 +53,6 @@ var (
 		metav1.NamespaceSystem,
 		metav1.NamespacePublic,
 	}
-	snapshotAnnotation = "snapshot.alpha.kubernetes.io/snapshot"
```

Comment:
Instead of removing it, we can comment variable declaration line. We may use this variable in future. Currently, code which uses this variable was [commented](https://github.com/openebs/cstor-operators/blob/master/pkg/webhook/pvc.go#L166-L169).